### ISSUE TITLE
JAWS reads garbage data '<voice name="PCCursorVoice"></voice>' with aria-live region inside a Table

### ISSUE THREAD

**Issue by kevin-dhaduk** [FreedomScientific/standards-support#687]
### Summary

> If an aria-live region is implemented inside a Table, whenever the aria-live region is updated - JAWS announces: 
>> '\<voice name="PCCursorVoice">Column Header\</voice>\<voice name="PCCursorVoice">Row Header\</voice> Updated Cell Data'
> 
> Example:
> 1. Suppose Column header is **Content** an Row header is **polite** of the live region cell that has value **1**
> 2. Do some action to update the live region (suppose the value is **1 2** now) and listen to JAWS announcements
> 3. JAWS announces: '\<voice name="PCCursorVoice">Content\</voice>\<voice name="PCCursorVoice">polite\</voice> 1 2'

### Expected result

> Not sure what can be the best behavior but I think JAWS should not announce \<voice> tags
> Expected JAWS announcement can be: 'Column Header - Row Header - Updated Cell Data'

### Actual result

> JAWS announces: 
>> '\<voice name="PCCursorVoice">Column Header\</voice>\<voice name="PCCursorVoice">Row Header\</voice> Updated Cell Data'

### Example

> 1. Go to https://codepen.io/kevindhaduk/pen/qBybrgW
> 2. Go to any Update button inside a Table and press enter while on Update button and listen to JAWS announcements

#### JAWS version and build number

>Tested on each of the following JAWS builds:
>>2022.2204.20
>>2022.2211.7
>>2023.2212.23
 
#### Operating System and version

> Windows 10 - Version 21H1 (OS Build 19043.928)

#### Browser and version:

> Chrome 108.0.5359.125
> Microsoft Edge 108.0.1462.54

#### Attachments

![JAWS aria-live inside table screenshot](https://user-images.githubusercontent.com/86653322/209718302-4ad9c4ca-43cb-4941-8859-acef00a6ebce.png)

https://user-images.githubusercontent.com/86653322/209719199-a812c35b-f809-4b2c-a442-564f2337e4c3.mp4

**Comment by JAWS-test** [comment]
The problem also occurs with older JAWS versions (for example, 2020 and 2021). I can also confirm the problem for JAWS 2022 and 2023. The problem occurs only because the live region is inside a table. It does not occur, for example, if the table is removed or marked with role=presentation. Since this is a serious error, I recommend a quick fix

**Comment by stevefaulkner** [comment]
@RS071459 I have raised it with the JAWS software development team, will update you when I get a response.

**Comment by RS071459** [comment]
> @RS071459 I have raised it with the JAWS software development team, will update you when I get a response.

Have you received any update from the JAWS development team?

**Comment by JustinK77** [comment]
Bumping this issue as it's we're running into this issue as well.

**Comment by tay1orjones** [comment]
We're also running into this in the Carbon Design System using JAWS 2023

https://react.carbondesignsystem.com/?path=/story/components-datatable-dynamic--default

**Comment by tay1orjones** [comment]
I emailed them per the [readme](https://github.com/FreedomScientific/standards-support/blob/806d9a787d06f3f5d36e34c94ecfc7bc521b1a87/README.md?plain=1#L21): 

> Note: please do not file general support issues here. Please [contact Freedom Scientific](https://www.freedomscientific.com/About/ContactUs) to report general issues with JAWS.

Here's the response I recieved:

> I show this issue is still being looked at. The bug number is: 134840. I don't have a time frame yet on a fix.

**Comment by stevefaulkner** [comment]
@tay1orjones I had a look at the bug, can reproduce issue. I am following the internal bug. Will let you know if any updates to it occur.

**Comment by BrettLewisVispero** [comment]
Can't reproduce in JAWS 2024
